src/style.css

```diff
@@ -1 +1,28 @@
-/* add styles here */
+* {
+  margin: 0;
+}
+
+html {
+  font-size: 10px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 1.6rem;
+  height: 1.6rem;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Here you are adding a right margin, which could potentially lead to issues when combined with other elements' margins. It's typically better to stick to a single direction for margins, such as only top or only bottom, to avoid unexpected layout behaviors.